src/components/AddItem/AddItem.jsx

```diff
@@ -1,5 +1,14 @@
+import React from 'react';
+
+import { Form } from '../Form/Form';
+import('./addItems.css');
+
 const AddItem = () => {
```

Comment:
what do you think if remove {} and put of this way:
const AddItem = () =>(<><Form /></>)

Reply:
Thank you @FauricioValencia We will work on it 😊